faq_bot/requirements.txt

```diff
@@ -0,0 +1,3 @@
+dff[telegram, tests] @ git+https://github.com/deeppavlov/dialog_flow_framework.git@merge/telegram
+sentence_transformers
+numpy
```

Comment:
Do we need numpy? It can be installed with sentence_transformers, does it?

Reply:
https://github.com/deeppavlov/dialog_flow_demo/pull/1/commits/b8243a998a832e98b6c0a584430089633149c79d